03-viz.Rmd

```diff
@@ -1,40 +1,528 @@
+
+1) Establish the setting and scope, and motivate why you did what you did. 
+2) Pose the question that your visualization answers. Justify why the question is important to answer.
+3) Answer the question using your visualization. Make sure you describe *all* aspects of the visualization (including describing the axes). But you 
+   can emphasize different aspects based on what is important to answering your question:
+    - **trends (lines):** Does a line describe the trend well? If so, the trend is *linear*, and if not, the trend is *nonlinear*. Is the trend increasing, decreasing, or neither?
+                        Is there a periodic oscillation (wiggle) in the trend? Is the trend noisy (does the line "jump around" a lot) or smooth?
+    - **distributions (scatters, histograms):** How spread out is the distribution? Where is it centered, roughly? Are there any obvious "clusters" or "subgroups", which would be visible as multiple bumps in the histogram?  
+    - **distributions of two variables (scatters):** is there a clear / strong relationship between the variables (points fall in a distinct pattern), a weak one (points fall in a pattern but there is some noise), or no discernible
+      relationship (the data are too noisy to make any conclusion)?
+    - **amounts (bars):** How large are the bars relative to one another? Are there patterns in different groups of bars? 
+4) Summarize your findings, and use them to motivate whatever you will discuss next.
+
+Below are two examples of how might one take these four steps in describing the example visualizations that appeared earlier in this chapter.
+Each of the steps is denoted by its numeral in parentheses, e.g. (3).
+
+**Mauna Loa Atmospheric CO2 Measurements:** (1) Many current forms of energy generation and conversion---from automotive engines to natural gas power plants---rely on burning
+fossil fuels and produce greenhouse gases, typically primarily carbon dioxide (CO2), as a byproduct. Too much of these gases in the Earth's atmosphere will cause it to trap
+more heat from the sun, leading to global warming. (2) In order to assess how quickly the atmospheric concentration of CO2 is increasing over time, we (3) used a dataset from
+the Mauna Loa observatory from Hawaii, consisting of CO2 measurements from 1959 to the present. We plotted the measured concentration of CO2 (on the vertical axis) over time
+(on the horizontal axis). From this plot you can see a clear, increasing, and generally linear trend over time. There is also a periodic oscillation that occurs once per year and
+aligns with Hawaii's seasons, with an amplitude that is small relative to the growth in the overall trend. This shows that atmospheric CO2 is clearly increasing over 
+time, and (4) it is perhaps worth investigating more into the causes.
+
+**Michelson Light Speed Experiments:** (1) Our modern understanding of the physics of light has advanced significantly from the 
```

Comment:
Same goes here, the reader cannot remember the plot at this point in the chapter.